sass/_themes.scss

```diff
@@ -106,6 +106,8 @@ $COLOR_CORAL_RED  : #ff444f;
 $COLOR_SKY_BLUE   : #2196f3;
 $COLOR_WHITE      : #ffffff;
 $COLOR_BLUE       : #1c5ae3;
+$CHART_BASELINE_DOWN: rgba(249, 84, 84, 0.32);
+$CHART_BASELINE_UP: rgba(76, 175, 80, 0.32);
```

Comment:
As mentioned in the design, we should use `0.32` opacity version of original colors, so why need to add new colors? we already have chart types original colors

Reply:
These 2 are not in theme colors. They're defined at the very first of `chart.scss` and are used as base colors in the chart.
Right?

Reply:
well, for candle type we have the colors we already know, for line chart we have `ChartMountainBg` and `ChartMountainBorder` prop in our theme palate, so as UI/UX defines that colors should be same but with `0.32` opacity for close. so better to change them if line color changed, and for closing, just convert them to `0.32` opacity. so no more new variable for them. am I right?

Reply:
also for candle, we have `ChartClosedCandleColor` in theme palate